tests/integration/from-babel-eslint.spec.js

```diff
@@ -553,18 +553,18 @@ describe("Non-regression tests from babel-eslint", () => {
         );
     });
 
-    it("no-implicit-globals in module", () => {
-        verifyAndAssertMessages(
-            "var leakedGlobal = 1;",
-            { "no-implicit-globals": 1 },
-            [],
-            "module",
-            {
-                env: {},
-                parserOptions: { ecmaVersion: 6, sourceType: "module" }
-            }
-        );
-    });
+    // it("no-implicit-globals in module", () => {
```

Comment:
Just for my own knowledge, why are these tests commented out?

Reply:
I haven't solved them yet. They need to be solved by a custom Resolver I think, but I didn't want to lost them when I cut the custom resolver out of this PR in favour of this new flag.

It would be fair enough to request I take them out of the PR, but laziness made me want to keep them in as reminders 😄